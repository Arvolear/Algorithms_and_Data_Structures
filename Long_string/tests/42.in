4082644901596319589289158363
50581965475736569438