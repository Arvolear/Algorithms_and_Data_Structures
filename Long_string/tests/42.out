4082644952178285065025727801
4082644851014354113552588925
206508203462236961313212754531824623619927909994
80713449